namui-animation-editor/src/graph_editor/graph_window/render/mod.rs

```diff
@@ -0,0 +1,270 @@
+mod render_graph;
+
+use std::fmt::Display;
+
+use super::*;
+use namui::animation::KeyframeValue;
+use render_graph::*;
```

Comment:
```suggestion
mod render_graph;

use super::*;
use namui::animation::KeyframeValue;
use render_graph::*;
use std::fmt::Display;
```

This is a result of gouping linting imports